randomizer/Patching/CosmeticColors.py

```diff
@@ -2532,7 +2557,18 @@ def placeKrushaHead(slot):
     kong_face_textures = [[0x27C, 0x27B], [0x279, 0x27A], [0x277, 0x278], [0x276, 0x275], [0x273, 0x274]]
```

Comment:
Since the krusha heads now are being based on color, I'd add a check here for colorblind mode being enabled. If it is, then return out of the function early to prevent the heads being overwritten.

Reply:
Good idea, return added